.github/workflows/version-updater.yml

```diff
@@ -34,7 +34,8 @@ jobs:
           git commit -am 'Update versioning in file from ${{ github.event.inputs.old_version_number }} to ${{ github.event.inputs.new_version_number }}'
           git push --set-upstream origin updater-job/update-to-${{ github.event.inputs.new_version_number }}
       - name: Raise a Pull-Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          sudo apt-get install
-          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+          sudo apt-get install gh
```

Comment:
Does this actually work on these instances? This isn't how `gh` is installed according to the official [documentation](https://github.com/cli/cli/blob/trunk/docs/install_linux.md), and running this on my vm results in "E: Unable to locate package gh"

Reply:
```suggestion
          sudo apt-get install gitsome
```

Reply:
Does not look like this has been addressed?

Reply:
Made changes to follow official documentation.